src/main/java/sc/fiji/bdvpg/bdv/navigate/ViewerTransformSyncStarter.java

```diff
@@ -0,0 +1,90 @@
+package sc.fiji.bdvpg.bdv.navigate;
+
+import bdv.util.BdvHandle;
+import net.imglib2.realtransform.AffineTransform3D;
+import net.imglib2.ui.TransformListener;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Action which synchronizes the display location of n BdvHandle
+ *
+ * Works in combination with the action ViewerTransformSyncStopper
+ *
+ * See also ViewTransformSynchronizationDemo
+ *
+ * author Nicolas Chiaruttini, BIOP EPFL, nicolas.chiaruttini@epfl.ch
+ *
+ */
+
+public class ViewerTransformSyncStarter implements Runnable {
+
+    BdvHandle[] bdvhs;
+
+    BdvHandle originatingBdvHandle = null;
+
+    Map<BdvHandle, TransformListener<AffineTransform3D>> bdvToTransformListener = new HashMap<>();
+
+    public ViewerTransformSyncStarter(BdvHandle[] bdvhs) {
+       this.bdvhs = bdvhs;
+    }
+
+    public void setOriginatingBdvHandle(BdvHandle bdvh) {
+        originatingBdvHandle = bdvh;
+    }
+
+    @Override
+    public void run() {
+
+        // Building circularly linked listeners with stop condition when all transforms are equal
+
+        AffineTransform3D at3Dorigin = null;
+
+        for (int i=0;i<bdvhs.length;i++) {
+            BdvHandle bdvhi = bdvhs[i];
+            if (bdvhi.equals(originatingBdvHandle)) {
```

Comment:
Would it be ok to use less abbreviations for object names?
For example,
- `bdvhi` => `bdvHandle`
- `bdvhs` => `bdvHandles`

For me, it would be much easier to read and review the code.

Reply:
oki doki! done in https://github.com/bigdataviewer/bigdataviewer-playground/pull/68/commits/72739e1cf201b698fe4dbfb38b982d5515b2e1a4

bdvhi is `currentBdvHandle`